src/components/ViewDao/ViewDaoContainer.tsx

```diff
@@ -1,17 +1,12 @@
 import { IDAOState } from "@daostack/client";
 import * as profilesActions from "actions/profilesActions";
-import * as uiActions from "actions/uiActions";
 import { getArc } from "arc";
-import * as classNames from "classnames";
 import CreateProposalContainer from "components/CreateProposal/CreateProposalContainer";
 import ProposalDetailsContainer from "components/Proposal/ProposalDetailsContainer";
 import Subscribe, { IObservableState } from "components/Shared/Subscribe";
-import * as appCss from "layouts/App.scss";
-import Util from "lib/util";
 import * as React from "react";
 import { BreadcrumbsItem } from "react-breadcrumbs-dynamic";
 import { Cookies, withCookies } from "react-cookie";
-import Joyride from "react-joyride";
```

Comment:
i hope you removed the tour stuff in a single commit so we can revert it when we want to bring it back :)